### PR TITLE
Update queue, using openpitrix.io/libqueue.

### PR DESCRIPTION
**Pull request by Ma-Dan**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -28,6 +29,7 @@ require (
 	github.com/speps/go-hashids v2.0.0+incompatible
 	github.com/stretchr/testify v1.3.0
 	github.com/ugorji/go v1.1.4 // indirect
+	go.etcd.io/etcd v3.3.12+incompatible
```
Comment (huojiao2006) [inline go.mod]: why there 2 different etcd package version?

File: pkg/services/executor/alert_receiver.go [hunk 2]
```diff
@@ -1,50 +1,42 @@
...
-
-	for i := 0; i < queueNum; i++ {
-		alertQueue = append(alertQueue, global.GetInstance().GetEtcd().NewQueue(fmt.Sprintf("%s-%d", constants.AlertTopicPrefix, i)))
+	c, err := q.New(queueType, queueConfigMap)
+	if err != nil {
+		logger.Error(nil, "Failed to connect redis queue: %+v.", err)
 	}
 
+	alertQueue, _ = c.SetTopic(constants.AlertTopicPrefix)
+
```
Comment (huojiao2006) [inline pkg/services/executor/alert_receiver.go:39]: need to deal with the error

**Review by huojiao2006 — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Ma-Dan) [reply go.mod]: go.etcd.io/etcd v3.3.12+incompatible removed.

File: pkg/config/config.go [hunk 3]
```diff
@@ -32,7 +32,11 @@ type Config struct {
 
 	Etcd struct {
```
Comment (huojiao2006) [inline pkg/config/config.go:33]: change the default name to ks etcd?

File: pkg/services/watcher/health_checker.go [hunk 4]
```diff
@@ -86,7 +86,7 @@ func (hc *HealthChecker) processTimeoutUpdatingAlerts() {
 	for _, alert := range alerts {
 		err := rs.UpdateAlertByAlertId(alert.AlertId, "updating", "adding")
```
Comment (huojiao2006) [inline pkg/services/watcher/health_checker.go]: using as constant for these fixed value paramters?

**Pull request merged**